src/styles/ForecastSummary.css

```diff
@@ -1,11 +0,0 @@
-.forecast-summary {
-  text-align: center;
-  padding: 32px;
-  background-color: whitesmoke;
-  border-style: solid;
-  border-radius: 4px;
```

Comment:
I would have a slightly bigger border radius as it isn't very noticeable. Maybe 8px